mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -63,10 +63,10 @@ def ocn_modelvsobs(config, field):
     if field.lower() == 'mld':
         obs_filename = "%s/holtetalley_mld_climatology.nc" % obsdir
 
-        ds = xr.open_mfdataset(infiles, preprocess=lambda x: preprocess_mpas(x, yearoffset=yr_offset,
-					   timeSeriesStats=True,
-                               timestr='time_avg_daysSinceStartOfSim',
-                               onlyvars=['time_avg_dThreshMLD']))
+        ds = xr.open_mfdataset(infiles, preprocess=lambda x: 
+                               preprocess_mpas(x, yearoffset=yr_offset,
+                                               timestr=['xtime_start', 'xtime_end'],
+                                               onlyvars=['time_avg_dThreshMLD']))
         ds = remove_repeated_time_index(ds)
```

Comment:
I realize that splitting the `lambda x:` and `process_mpas(...` lines here and elsewhere is not optimal.  I have not found a more elegant solution that is PEP8 compliant.